hilfe/handbuch/dienstplan/index.md

```diff
@@ -1,43 +1,45 @@
 ---
+layout: layouts/hilfe/single.njk
 keywords: []
 title: Dienstplan
 weight: 1
 icon: table
+anchors: ["testing", "testing2"]
```

Comment:
Here's a simple example I was trying to pull, though these aren't "correct" anchors (they'd be a dictionary/object with `hash` and `title`, as per other articles).